just_audio_background/lib/just_audio_background.dart

```diff
@@ -443,20 +436,18 @@ class _PlayerAudioHandler extends BaseAudioHandler
     return LoadResponse(duration: response.duration);
   }
 
-  Future<SetVolumeResponse> customSetVolume(SetVolumeRequest request) async {
-    return await (await _player).setVolume(request);
-  }
+  Future<SetVolumeResponse> customSetVolume(SetVolumeRequest request) async =>
+      (await _player).setVolume(request);
 
-  Future<SeekResponse> customPlayerSeek(SeekRequest request) async {
-    return await (await _player).seek(request);
-  }
+  Future<SeekResponse> customPlayerSeek(SeekRequest request) async =>
+      (await _player).seek(request);
 
   Future<SetShuffleOrderResponse> customSetShuffleOrder(
       SetShuffleOrderRequest request) async {
     _source = request.audioSourceMessage;
     _updateShuffleIndices();
     _broadcastStateIfActive();
-    return await (await _player).setShuffleOrder(SetShuffleOrderRequest(
+    return (await _player).setShuffleOrder(SetShuffleOrderRequest(
```

Comment:
Does the lint require this change? When a method is declared as `async`, I prefer to await all futures in the body.

Reply:
Almost certainly these would compile the same way. But for consistency, I prefer to always use await inside of an `async` method.

Reply:
Oh, its actually due to the lint - adding `await` back raises [unnecessary_await_in_return](https://dart-lang.github.io/linter/lints/unnecessary_await_in_return.html) should I ignore and add await back or it's ok as it is now?

Reply:
It sounds like unnecessary_await_in_return is something I wouldn't want to adopt. Treating return types as lax (might be a future, might not be a future) doesn't sit well with me.

Reply:
Ok then I'll return `await`.